assets/js/modules/analytics-4/datastore/audiences.js

```diff
@@ -682,12 +685,93 @@ const baseSelectors = {
 				return undefined;
 			}
 
-			const audiencesUserCountReportOptions =
-				getAudiencesUserCountReportOptions( configurableAudiences );
+			// eslint-disable-next-line @wordpress/no-unused-vars-before-return
+			const [ siteKitAudiences, otherAudiences ] =
+				getConfiguredSiteKitAndOtherAudiences();
+
+			const isSiteKitAudiencePartialData =
+				hasAudiencePartialData( siteKitAudiences );
+
+			if ( undefined === isSiteKitAudiencePartialData ) {
+				return undefined;
+			}
+
+			const siteKitUserCountReportError = isSiteKitAudiencePartialData
+				? getErrorForSelector( 'getReport', [
+						getSiteKitAudiencesUserCountReportOptions(),
+				  ] )
+				: undefined;
+
+			const otherUserCountReportError =
+				! isSiteKitAudiencePartialData || otherAudiences?.length !== 0
+					? getErrorForSelector( 'getReport', [
+							getAudiencesUserCountReportOptions(
+								isSiteKitAudiencePartialData
+									? otherAudiences
+									: configurableAudiences
+							),
+					  ] )
+					: undefined;
+
+			return [ otherUserCountReportError, siteKitUserCountReportError ];
```

Comment:
Hmm, sorry to ask this because I realise this is how we've defined it in the IB, but please can you swap the order of these returned errors?

It looks a bit odd having the `getConfiguredSiteKitAndOtherAudiences()` returning `[ siteKitAudiences, otherAudiences ]` while this returns the errors in reverse order.

It would be more consistent to return `[ siteKitUserCountReportError, otherUserCountReportError  ]`. Please can you update accordingly?